src/TwigEnvironmentFactory.php

```diff
@@ -110,7 +111,7 @@ public function __invoke(ContainerInterface $container) : TwigEnvironment
             }
             try {
                 $timezone = new DateTimeZone($timezone);
-            } catch (\Exception $e) {
+            } catch (Throwable $e) {
```

Comment:
This change is not safe as it changes behavior and as such it is *not up to a CS tool to decide*. Intercepting exceptions is not quite the same as intercepting catchable php errors.

Reply:
In this case you actually want to catch everything. But most cases you catch specific component exceptions like `Zend\Expressive\Exception\RuntimeException`. 

For the few exceptions the sniff can be disabled. Disabling sniffs is documented and so are the sniffs.